from django.core.exceptions import ValidationError
from django.test import TestCase
from .models import User

# Create your tests here.
